lib/transmuxer.js

```diff
@@ -1055,6 +1055,11 @@ collectDTSInfo = function (track, data) {
   } else {
     track.minSegmentDTS = Math.min(track.minSegmentDTS, data.dts);
   }
+  if (track.minSegmentPts === undefined) {
```

Comment:
Might as well change the min/maxSegmentDTS names to min/maxSegmentDts while you are at it here just for consistency's sake.